test/fixtures/index.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
   inputWebPWithTransparency: getPath('5_webp_a.webp'), // http://www.gstatic.com/webp/gallery3/5_webp_a.webp
   inputTiff: getPath('G31D.TIF'), // http://www.fileformat.info/format/tiff/sample/e6c9a6e5253348f4aef6d17b534360ab/index.htm
   inputTiffCielab: getPath('cielab-dagams.tiff'), // https://github.com/lovell/sharp/issues/646
+  inputTiffUncompressed: getPath('uncompressed_tiff.tiff'),
```

Comment:
https://code.google.com/archive/p/imagetestsuite/wikis/TIFFTestSuite.wiki includes 20 images tagged as "Compression Scheme: None" so might provide something suitable. Your own image is fine as long as you grant us permission to distribute it :)